sktime/classification/distance_based/_time_series_neighbors.py

```diff
@@ -225,8 +203,7 @@ def kneighbors(self, X, n_neighbors=None, return_distance=True):
 
         Parameters
         ----------
-        X : sktime-format pandas dataframe with shape([n_cases,n_dimensions]),
-        or numpy ndarray with shape([n_cases,n_readings,n_dimensions])
+        X : sktime-compatible data format, Panel or Series, with n_samples series
         y : {array-like, sparse matrix}
             Target values of shape = [n_samples]
```

Comment:
I dont think we want any `y` in docstring

Reply:
ah, I just copy-pasted that. You are right, though.